app/models/visorium.rb

```diff
@@ -0,0 +1,3 @@
+class Visorium < ApplicationRecord
+	validates :date, :place, :schedule, :club, :description, :category, presence: true	
```

Comment:
Fix this indentation, I can see you are using two spaces in most of the files, please keep that consistency 🤓